### ISSUE TITLE
Installation not working

### ISSUE THREAD

**Issue by hugues1277** [givanz/Vvveb#199]
Hello

I encountered some errors during the installation, moreover the installation does not work. too bad

your function create an error
Vvveb\System\Core\Response::getType()
- public function getType($type) =>  getType()

after install all in my server then whith docker (I commented # COPY php.ini ${PHP_INI_DIR} in Dockerfile which not working with nginx) 
It still not working : 

Page not found!
Html template not found!
/var/www/html/public/themes/landing/index.html on line 0

It's a shame, because I imagine I'll find a really great alternative to WordPress.
By the way, since your code is composed of PHP and nginx, Yunohost?

**Comment by git-gnulf** [comment]
Yea! This would be great! I tried to run vvveb with YunoHost. I used the "my_webapp"-App, but I fight with the nginx-conf-rules ... and I lose 😉 It s a bit complex for me. But someone knowing nginx and vvveb can build the Yunohost-App easy ... I hope! I m not sure. But if someone from your team could do this, I will be happy - I m sure 😉

**Comment by hugues1277** [comment]
Well, there must have been updates because I just installed it on my yunohost
- creation of a myWebApp app
- unzip your app
- installation with a database, everything went well

**Comment by git-gnulf** [comment]
Oh, that sounds good. Have you configured some special lines in /etc/nginx/conf.d/my_webapp.d/my_webapp.conf? Of cource the alias-line - with the addendum "public/" ...

**Comment by hugues1277** [comment]
No, just installed php8.3-curl, but it's strange that it wasn't already installed
